packages/wdio-browserstack-service/src/service.js

```diff
@@ -6,10 +6,12 @@ import { BROWSER_DESCRIPTION } from './constants'
 const log = logger('@wdio/browserstack-service')
 
 export default class BrowserstackService {
-    constructor (options, caps, config) {
+    constructor ({ preferScenarioName = false }, caps, config) {
         this.config = config
-        this.failures = 0
         this.sessionBaseUrl = 'https://api.browserstack.com/automate/sessions'
+        this.preferScenarioName = preferScenarioName
```

Comment:
This seems to be a boolean

```suggestion
        this.preferScenarioName = Boolean(preferScenarioName)
```